src/pages/index.tsx

```diff
@@ -49,10 +50,17 @@ const Home: NextPage<any> = () => {
   const [modalUser, setModalUser] = useState<PublicUser | null>(null);
   const [modalType, setModalType] = useState<string>("connect");
   const [sidebarState, setSidebarState] = useState<HeaderOptions>("explore");
+  const [startingRequestsTab, setStartingRequestsTab] = useState<0 | 1>(0);
```

Comment:
Why are we using numbers here instead of a boolean. Is this to allow future extensibility for new tabs? If so maybe we could use a string enum type instead.